testdata/storage/iscsi/service.json

```diff
@@ -0,0 +1,20 @@
+{
+    "kind": "Service",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "iscsi-target"
+    },
+    "spec": {
+	"selector": {
```

Comment:
Would you mind to fix the indentions ? It's hard to read.

Reply:
Good catch! Updated.